src/styles/mixins.module.scss

```diff
@@ -23,3 +33,40 @@
   font-family: 'OpenSans Regular';
   color: variables[black];
 }
+
+@mixin openSansMedium {
+  font-family: 'OpenSans Extra Medium';
+  color: #231f20;
+}
+
+@mixin openSansSemiBold {
+  font-family: 'OpenSans Extra Semi Bold';
+  color: #231f20;
+}
+
+@mixin montserratMedium {
+  font-family: 'Montserrat Medium';
+  color: #231f20;
+}
+
+@mixin padding {
```

Comment:
Since we are using this globally, can we name it something that communicates that? Some like pagePadding or pageSpacing or pageWrapper, something just a little more specific.